etc/kayobe/pulp.yml

```diff
@@ -4,7 +4,7 @@
 
 # Base URL of the local Pulp service.
 # Default uses the seed node's IP on the admin network.
-pulp_url: "{{ 'https' if pulp_enable_tls | bool else 'http' }}://{{ admin_oc_net_name | net_ip(groups['seed'][0]) }}:{{ pulp_port }}"
+pulp_url: "{{ 'https' if pulp_enable_tls | bool else 'http' }}://{{ hostvars[groups['seed'][0]].admin_oc_net_name }}:{{ pulp_port }}"
```

Comment:
This isn't the same thing - `admin_oc_net_name` is the string name of the admin overcloud network, not an IP address.

Reply:
Corrected